dali/python/bundle-wheel.sh

```diff
@@ -122,6 +122,8 @@ DEPS_LIST=(
     "${DEPS_PATH}/lib/libzstd.so.1"
     "${DEPS_PATH}/lib/libz.so.1"
     "${DEPS_PATH}/lib/libcfitsio.so.4"
+    "lib/libcvcuda.so.0"
```

Comment:
Do you know if there is any symbol versioning in CV-CUDA? In the case of FFmpeg we append `DALI_` prefix to symbols so they won't clash with the symbols from any other FFmpeg build.
I'm just afraid of what may happens when the user will do:
```import cvcuda
import nvidia.dali```
and I suspect that symbols from the first cvcuda are going to be used.